aspnetcore/blazor/file-downloads.md

```diff
@@ -41,7 +41,7 @@ When downloading files from a different origin than the app, Cross-Origin Resour
 
 ## Security considerations
 
-Use caution when providing users with the ability to download files from a server. Attackers may execute [Denial of Service (DoS)](/windows-hardware/drivers/ifs/denial-of-service) attacks, [API exploitation attacks](https://developer.mozilla.org/docs/Web/HTML/Element/a#security_and_privacy), or attempt to compromise networks and servers in other ways.
+Use caution when providing users with the ability to download files from a server. Cyberattackers may execute [Denial of Service (DoS)](/windows-hardware/drivers/ifs/denial-of-service) attacks, [API exploitation attacks](https://developer.mozilla.org/docs/Web/HTML/Element/a#security_and_privacy), or attempt to compromise networks and servers in other ways.
```

Comment:
```suggestion
Use caution when providing users with the ability to download files from a server. Cyber attackers may execute [Denial of Service (DoS)](/windows-hardware/drivers/ifs/denial-of-service) attacks, [API exploitation attacks](https://developer.mozilla.org/docs/Web/HTML/Element/a#security_and_privacy), or attempt to compromise networks and servers in other ways.
```

Reply:
The guidelines call for one word, no hyphen.

Reply:
Very minor and feel free to refuse or accept: I think both "cyberattackers"and "cyber attackers" are used in the development community, but I think "cyber attackers" might be used more often.

Reply:
I don't think we have a choice given how explicit they are in the style manual. We've broken with them a bit here and there, but I think AP and Chicago are in agreement. My Chicago is buried at the moment tho ... in a box somewhere around here from the last move 😆.